.qlty/qlty.toml

```diff
@@ -14,34 +14,11 @@
 config_version = "0"
 
 exclude_patterns = [
-    "*_min.*",
-    "*-min.*",
-    "*.min.*",
-    "**/.yarn/**",
-    "**/*.d.ts",
-    "**/assets/**",
-    "**/bower_components/**",
-    "**/build/**",
-    "**/cache/**",
-    "**/config/**",
-    "**/db/**",
-    "**/deps/**",
-    "**/dist/**",
-    "**/extern/**",
-    "**/external/**",
     "**/fixtures/**",
-    "**/generated/**",
-    "**/Godeps/**",
-    "**/gradlew/**",
-    "**/linters/**",
-    "**/mvnw/**",
-    "**/node_modules/**",
     "**/protos/**",
-    "**/seed/**",
-    "**/target/**",
-    "**/testdata/**",
-    "**/tests/lang/**",
-    "**/vendor/**",
+    "**/*.in/**",
+    "**/*.test.ts/**",
+    "plugins/**",
```

Comment:
Trim down exclude_patterns including removing some exclude patterns which were erroneously excluding real code (e.g. in folders with "build", "cache", or "config" in the names)